### PR TITLE
A network for learning visual graphs

### PR DESCRIPTION
**Pull request by jodoin**

### CODE REVIEW

File: deep-learning/_posts/2017-09-15-deepGraph.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+![](/deep-learning/images/GSNN/sc01.png)
+
+## Proposed network
+
+The goal of the system is to learn an adjacency matrix as well as the state of every node.  The adjacency matrix defines which visual nodes are connected and the state if the node is present in the image.  This is learned via a **propagation network**.  
+
+![](/deep-learning/images/GSNN/sc02.png)
+
+However, since learning the entier graph all at once is time expansive, they start by propagating information from visual nodes detected by Faster R-CNN.
```
Comment (ppoulin91) [inline deep-learning/_posts/2017-09-15-deepGraph.md]: Typo: expensive

Comment (jodoin) [reply deep-learning/_posts/2017-09-15-deepGraph.md]: done

File: deep-learning/_posts/2017-09-15-deepGraph.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    authors: "Kenneth Marino, Ruslan Salakhutdinov, Abhinav Gupta"
+    title:   "The More You Know: Using Knowledge Graphs for Image Classification"
+    venue:   "CVPR 2017"
+pdf: "https://arxiv.org/pdf/1612.04844.pdf"
+---
+
+## Summary
+
+This paper investigates the use of knowledge graphs and shows that using this knowledge improves performance
+on image classification.  The propose a  Graph Search Neural Network (GSNN) that reasons
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-15-deepGraph.md]: They propose a(*)Graph

* remove second space ;)

File: deep-learning/_posts/2017-09-15-deepGraph.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+![](/deep-learning/images/GSNN/sc01.png)
+
+## Proposed network
+
+The goal of the system is to learn an adjacency matrix as well as the state of every node.  The adjacency matrix defines which visual nodes are connected and the state if the node is present in the image.  This is learned via a **propagation network**.  
+
+![](/deep-learning/images/GSNN/sc02.png)
+
+However, since learning the entier graph all at once is time expensive, they start by propagating information from visual nodes detected by Faster R-CNN.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-15-deepGraph.md]: entire

Comment (jodoin) [reply deep-learning/_posts/2017-09-15-deepGraph.md]: nice! no one can fool Carl.

**Pull request merged**